README.md

```diff
@@ -63,10 +135,13 @@ Our [CONTRIBUTING](https://github.com/OpenLiberty/open-liberty/blob/master/CONTR
     
     ```open-liberty\dev\cnf\release\dev\openliberty\<version>\openliberty-xxx.zip```
 
-5. Go [Open issues](https://github.com/OpenLiberty/open-liberty/issues), [Review existing contributions](https://github.com/OpenLiberty/open-liberty/pulls), or [Submit fixes](https://github.com/OpenLiberty/open-liberty/blob/master/CONTRIBUTING.md).
+## License
+[![License](https://img.shields.io/badge/License-EPL%201.0-green.svg)](https://opensource.org/licenses/EPL-1.0)
```

Comment:
this badge is already listed up top

Reply:
ok yes, I saw some examples of sprinkling the badges throughout - but agreed its redundant.